src/main/java/com/mycompany/hangman/HangmanApp.java

```diff
@@ -25,6 +31,18 @@ public void start()
         GameOverView gov = new GameOverView(view, controller);
         controller.addView(gov);
 
+        ActionRepository actionRepo = new DefaultActionRepository();
+        {
+            actionRepo.add(new NewGameAction(controller));

```

Comment:
there’s no need to duplicate the knowledge of action types. leave this here but in GameMenu you should simply add a collection of action objects to the menu. the menu api adds menu items automayically when you add an action object. the gamemenu class should only need to reference action objects by name because it decides where they go on the menu. but it shouldnt have to know about their types.